recipes/server_setup.rb

```diff
@@ -114,37 +121,45 @@
       end
 
       # Create option string
+      force = false
       options = ''
       case volume_values['volume_type']
       when 'replicated'
-        # Ensure the trusted pool has the correct number of bricks available
-        if brick_count < volume_values['replica_count']
-          Chef::Log.warn("Correct number of bricks not available: #{brick_count} available, #{volume_values['replica_count']} required for volume #{volume_name}. Skipping...")
-          next
-        else
-          options = "replica #{volume_values['replica_count']}"
-          volume_bricks.each do |peer, vbricks|
-            options << " #{peer}:#{vbricks.first}"
-          end
-        end
+        # Replicated can be anything from a single node to X nodes. Replica_count should equal number of bricks.
```

Comment:
`Replica_count should equal number of bricks` means this should still be checked with the if statement that was removed above